lib/fizzy/cxx20/bit.hpp

```diff
@@ -24,3 +24,41 @@ template <class To, class From>
     return dst;
 }
 }  // namespace fizzy
+
+#ifdef __cpp_lib_bitops
+
+#include <bit>
+
+namespace fizzy
+{
+constexpr int popcount(uint32_t x) noexcept
+{
+    return std::popcount(x);
+}
+
+constexpr int popcount(uint64_t x) noexcept
+{
+    return std::popcount(x);
+}
+}  // namespace fizzy
+
+#else
+
+#include <cstdint>
```

Comment:
This is also needed for C++20 variant.

Reply:
Right. Will fix. Though it seems all the places this is used at already includes `cstdint`. See the `f` commit which has a warning to see if the code path is compiled (and it is).

Reply:
Just include it on the top of the file.

Reply:
Done.